src/main/kotlin/org/jetbrains/research/testspark/data/ProjectContext.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.research.testspark.data
+
+import com.intellij.openapi.module.Module
+import com.intellij.psi.PsiClass
+
+data class ProjectContext(
+    // The class path of the project.
+    var projectClassPath: String? = null,
+
+    // The URL of the file being tested.
+    var fileUrl: String? = null,
+
+    var cutPsiClass: PsiClass? = null,
+
+    var classFQN: String? = null,
+
```

Comment:
I think it is better to have a comment above with the extension of the abbreviation `FQN`.